universal-application-tool-0.0.1/conf/messages

```diff
@@ -0,0 +1,28 @@
+# This is the default language for the application. Please provide context for each message for use in translations.
+# If a translation cannot be found for a given language, it will default to this file's text instead.
+
+#---------------------------------------------------------------------#
+# COMPONENTS - contains text used in HTML components such as buttons. #
+#---------------------------------------------------------------------#
+
+# The text on the button an applicant clicks to apply to a specific program.
+button.apply=Apply
+
+# The button text for saving the current applicant-entered data and continuing to the next block in the form.
+button.nextBlock=Save and continue
+
+# A toast message that displays when a program is not fully localized to the applicant's preferred locale.
+toast.localeNotSupported=We're sorry, this program is not fully translated to English.
```

Comment:
suggestion: Since this is the default/fallback message, maybe we make this message more generic, like "We're sorry, this program is not fully translated to the selected language."

Reply:
Good catch! I had copied this from English, let me update

Reply:
Updated to "We're sorry, this program is not fully translated to your preferred language."